pkg/driver/controller.go

```diff
@@ -159,8 +160,7 @@ func sendEventOrIgnore(pvcName, pvName, capacity, method string) {
 		analytics.New().CommonBuild(DefaultCASType).ApplicationBuilder().
 			SetVolumeName(pvName).
 			SetVolumeClaimName(pvcName).
-			SetLabel(analytics.EventLabelCapacity).
```

Comment:
This bit is removed... why?

Reply:
The v0.1.0 version of the google-analytics-4 module used fields Category, Action, Label and Value to contain parameters for node-count (during install event), replica-count and capacity (during provision and deprovision events). These containers are inconvenient to look up on the Google Analytics dashboard.

v0.2.0 has replaced these containers in favour of new fields node_count, replica_count, vol_capacity. These contain the same values as before but they are visible alongside the usual metrics... Node OS, Engine Version, K8s version, etc.

Ref:
https://github.com/openebs/google-analytics-4/pull/8